django_project/cplus_api/utils/worker_analysis.py

```diff
@@ -298,7 +299,20 @@ def create_and_upload_output_layer(
         scenario=scenario_task,
         group=group
     )
-    with open(file_path, 'rb') as output_file:
+    cog_name = (
```

Comment:
@zamuzakki please also update the filename and filesize in the output_layer.
https://github.com/kartoza/cplus-api/blob/fa88fd5a09c7a771fd79e5748abe02fa00babedc/django_project/cplus_api/utils/worker_analysis.py#L293

Reply:
sorry, just the filesize, it seems that the filename is not changed, right?

Reply:
The filename is not changed. I'll update the size.